google-cloud-storage/test/google/cloud/storage/retry/conformance_test.rb

```diff
@@ -0,0 +1,165 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "helper"
+require "net/http"
+require_relative "../../../../../conformance/v1/proto/google/cloud/conformance/storage/v1/tests_pb"
+require_relative "./utils.rb"
+
+Google::Apis.logger.level = Logger::DEBUG
```

Comment:
Yes, I'm wary of this, especially since it's being done globally. Are there tests that assert against log output? Otherwise, I'd rather we didn't do this.